store/file_store_test.go

```diff
@@ -182,6 +182,22 @@ func TestCreate(t *testing.T) {
 	if exists(root, scratchdir, "abc") {
 		t.Errorf("File scratch abc exists")
 	}
+
+	_, err = s.Create("abc/abc")
+	if err != ErrKeyContainsSlash {
+		t.Errorf("Received error %s", err.Error())
+	}
+
+	_, err = s.Create("def abc")
+	if err != ErrKeyContainsWhiteSpace {
+		t.Errorf("Received error %s", err.Error())
+	}
+
+	_, err = s.Create("defabc")
+	if err != ErrKeyContainsControlChar {
+		t.Errorf("Received error %s", err.Error())
+	}
+
```

Comment:
Move these into a new test function `TestInvalidKeys`, and use a table to hold the test cases. Like https://github.com/ndlib/bendo/blob/master/store/file_store_test.go#L13 does.